packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py

```diff
@@ -60,17 +61,13 @@ async def run_dynamic_service(
 async def stop_dynamic_service(
     rabbitmq_rpc_client: RabbitMQRPCClient,
     *,
-    node_id: NodeID,
-    simcore_user_agent: str,
-    save_state: bool,
+    rpc_dynamic_service_stop: RPCDynamicServiceStop,
```

Comment:
Suggestion: I don't think we care about the technology you use for stopping services. So I would suggest to remove RPC from the names here.

Reply:
and actually same for creation

Reply:
renamed both.